### PR TITLE
pubsub/doc: delete obsolete sentence about Message.CanNack

### PR DESCRIPTION
**Pull request by Rolinh**
CanNack() is a method of Subscription, not Message.

### CODE REVIEW

File: pubsub/pubsub.go [hunk 1]
```diff
@@ -76,8 +76,8 @@
 //
 // Since Message.Nack never makes sense for some providers (for example, for
 // at-most-once providers, the provider can't redeliver the message), Nack will
-// panic if called for some providers. You can call Message.CanNack to see if
-// it is available.
+// panic if called for some providers. You can call Subscription.CanNack to see
```
Comment (vangent) [inline pubsub/pubsub.go]: Thanks for catching this! Actually, this sentence should be deleted entirely. `Subscription.CanNack` is on the driver interface type (`driver.Subscription`, not `pubsub.Subscription`), so it is not exposed to end users. We are considering exposing it to end users via `Message.Nackable` (currently commented out below).

So, do you mind just deleting the sentence starting with "You can call"?

Comment (Rolinh) [reply pubsub/pubsub.go]: Sure. Done now.

File: pubsub/pubsub.go [hunk 2]
```diff
@@ -74,11 +74,6 @@
 //    ever call Message.Ack.
 //  - func() { log.Info("ack called!") }: log. Softer than panicking.
 //
```
Comment (vangent) [inline pubsub/pubsub.go]: Sorry, please only delete the last sentence from the paragraph, not the whole paragraph.

Comment (Rolinh) [reply pubsub/pubsub.go]: Oh yes. It was indeed clear, sorry about that...

**Pull request merged**